common/src/main/java/com/wynntils/wynn/model/questbook/QuestInfo.java

```diff
@@ -122,6 +140,14 @@ private static String getQuestName(ItemStack item) {
         return m.group(1);
     }
 
+    private static boolean isQuestTracked(ItemStack item) {
+        String rawName = item.getHoverName().getString();
```

Comment:
I'm thinking maybe QUEST_NAME_MATCHER from QuestBookManager could move to this class, and be used here? I don't really like when we have multiple ways of doing the same thing.

Reply:
You resolved this without doing anything? Did you disagree or just forgot? 

To clarify: my point is that I don't like that we both have a pattern and a `rawName.endsWith("§e[Tracked]")` that tries to do more or less the same thing, with the logic spread between two classes.

Reply:
I removed the duplicate pattern, I thought this was that suggestion. I will look into this tomorrow.

Reply:
I might start to confuse things and forget exactly what I requested as well. :) But in general I am wary of "almost-duplicated" matches (this was the cause of much grief in legacy), and this looked like one such instance.

Reply:
Yeah, but, how would I remove this duplication? Perhaps use the pattern here as well, and check if there is the tracked part. Since we use a lot of static methods for parsing, there is no simple way